docs/index.asciidoc

```diff
@@ -110,13 +110,48 @@ append and indirect cannot be combined and will fail validation. +
 ===============================
 
 *Delimiter repetition:* +
-In the source text if a field has variable width padded with delimiters, the padding will be ignored. +
-e.g. for texts of:
+Delimiter repetition in the source text can occur unintentionally or intentionally. +
+
+[NOTE]
+===============================
+From version 1.1.1 onwards, repeated delimiter handling has changed to handle
+the unintentional repetition case by default because this is far more common
+than the intentional repetition case. +
+*If you are already using Dissect AND your source text has intentional repetition,
+you will need to change your configs.*
+===============================
+
+**Unintentional repetition** +
+This is caused when the source generator writes empty values in between delimiters. A typical example
+of this is CSV style sections. +
+For a dissection of `%{name},%{addr1},%{addr2},%{addr3},%{city},%{zip}`; we are expecting 6 fields. +
```

Comment:
Use a comma instead of a semicolon after the introductory phrase. Semicolons are for connecting two independent clauses, but here you have a dependent clause followed by an independent clause.

Also we typically avoid saying "we" in the docs. See my comment earlier about structuring the examples because this will probably help you avoid saying "we" here.